src/Core/BoundedContexts/Tester/WB.Core.BoundedContexts.Tester/ViewModels/LoginViewModel.cs

```diff
@@ -15,7 +13,7 @@
 
 namespace WB.Core.BoundedContexts.Tester.ViewModels
 {
-    public class LoginViewModel : BaseViewModel
+    public class LoginViewModel : BaseAuthenticatedViewModel

```

Comment:
may be inherit from BaseViewModel

Reply:
Principal and ViewModelNavigationService are required. They are already in BaseAuthenticatedViewModel and otherwise should be stored here. All 3 LoginViewModel for all apps derive from the same BaseAuthenticatedViewModel that makes it easier to manage.